src/app/oauth/services/userinfo.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Observable } from 'rxjs/Observable';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable()
+export class UserInfoService {
+  constructor(private http: HttpClient) {}
+
+  public getUserInfo(accessToken): Observable<any> {
+    return this.http.post(`https://www.googleapis.com/oauth2/v3/userinfo?access_token=${accessToken}`, '');
```

Comment:
Should this URL be externalized in environment file?

Reply:
Forgot to move it to env file.. will make this change.